relay-sampling/src/lib.rs

```diff
@@ -427,57 +399,95 @@ impl SamplingRule {
     ///
```

Comment:
Let's update this doc comment.

Reply:
^ One line above, it says `Returns whether the sampling rule is active`. We should change it to "Returns an `ActiveRule` if the rule is active", or something like that.